HW03/maksim_manzhai/maksim_manzhai_hw_03_t_02.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+DATETIME = %r{\d{2}/\w{3}/\d{4}:\d{2}:\d{2}:\d{2}\s\+\d{4}}
+IP = /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/
+ADDRESS = %r{/\w*/\d*/\w*}
+
+class Homework3
+  def task2(logs)
+    return [] if logs.empty?
+
+    lines = logs.split(/\n/)
+
+    formated_lines = lines.find_all { |line| line.include?('POST') }
+    formated_lines.each { |line| return [] if wrong_format?(line) }
+    formated_lines.collect do |line|
```

Comment:
You don't need to overide formated_lines variable each time. Try to make it easier